specifyweb/frontend/js_src/lib/utils/cache/definitions.ts

```diff
@@ -75,6 +75,10 @@ export type CacheDefinitions = {
   } & {
     readonly /** Open nodes in a given tree */
     [key in `conformations${AnyTree['tableName']}`]: Conformations;
+  } & {
+    readonly isSplit: boolean;
+  } & {
+    readonly isHorizontal: boolean;
```

Comment:
```suggestion
  } & {
    readonly isSplit: boolean;
    readonly isHorizontal: boolean;
```

Reply:
you can't mix the `{ [key in ...]: ... }` and `{ key: ...}` styles in the same type at the same time, which is why I had `&` in there before

but you can have two `{ key: ...}` `{ key: ...}` in the same object without the need for `&`